core/src/main/java/org/apache/calcite/util/BuiltInMethod.java

```diff
@@ -443,6 +443,7 @@ public enum BuiltInMethod {
       String.class, boolean.class),
   REPLACE(SqlFunctions.class, "replace", String.class, String.class,
       String.class),
+  TRANSLATE(SqlFunctions.class, "translateWithCharset", String.class, String.class),
```

Comment:
> Nearly there. A few doc changes will be helpful, because these functions are confusingly similar to other TRANSLATE and CONVERT functions

@julianhyde  Thanks for the review. I've updated the docs.

Reply:
Done.